package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.40",
+  "version": "0.6.00",
```

Comment:
Double zero, wow!

Reply:
hahah oops

Reply:
Double `00` makes "cents" doesn't it?

😆 Good thing no one will see that joke..

Reply:
Augh.